### ISSUE TITLE
doesn't seem to be working with 16.08pre1

### ISSUE THREAD

**Issue by sljunkie** [manjaro/packages-extra#95]
I installed Manjaro GNOME 16.06 and the the utility worked as expected. With the upgrade to 16.08pre1 changing the icon theme and the cursor theme doesn't work.

I reported this here https://github.com/Nano77/gdm3setup/issues/39 but on my Arch Linux (where the AUR packages are used) box the utility works as expected, so I'm reporting here also.

**Comment by Ste74** [comment]
Do you mean with 16.08 pre1 ? I assume  the regular update no a new ISO. 
Il 31/lug/2016 23:26, Luis Henrique Mello <notifications@github.com> ha scritto:I installed Manjaro GNOME 16.06 and the the utility worked as expected. With the upgrade to 16.08pre1 changing the icon theme and the cursor theme doesn't work.

I reported this here Nano77/gdm3setup#39 but on my Arch Linux (where the AUR packages are used) box the utility works as expected, so I'm reporting here also.

**Comment by philmmanjaro** [comment]
@sljunkie: Download the AUR package and compile it on your box. If it works we have to repack our package, as we built it a while ago. See also the error messages in your terminal.

**Comment by sljunkie** [comment]
@philmmanjaro the AUR package didn't work either. Notice that on AUR it was flagged out of date; the reason given was "It doesn't work with the last GDM versions."

**Comment by Ste74** [comment]
Infact the our and the aur version is the same. I have upgrade the pkg for 16.06 version. In your arch box are up to date the system? 
Il 01/ago/2016 10:39, Luis Henrique Mello <notifications@github.com> ha scritto:@philmmanjaro the AUR package didn't work either. Notice that on AUR it was flagged out of date; the reason given was "It doesn't work with the last GDM versions."

**Comment by sljunkie** [comment]
it seems the tool itself needs work.

For instance I try to use `gdmshelltheme` which comes with the `gdm3setup-utils` package and I have this:

```
$ sudo gdmshelltheme -l
Arc
Arc-Dark
Vertex-Maia
$ sudo gdmshelltheme -d Arc
Traceback (most recent call last):
  File "/usr/bin/gdmshelltheme", line 84, in <module>
    Set_Shell_theme("Adwaita")
  File "/usr/bin/gdmshelltheme", line 22, in Set_Shell_theme
    theme = Get_Shell_theme()
  File "/usr/bin/gdmshelltheme", line 10, in Get_Shell_theme
    ifile = file('/usr/share/gnome-shell/theme/gnome-shell.css',"r")
IOError: [Errno 2] No such file or directory: '/usr/share/gnome-shell/theme/gnome-shell.css'
```

**Comment by Ste74** [comment]
Have you try to switch on adwaita ?

**Comment by sljunkie** [comment]
Yes, to no avail.

How do I change the icon theme and the cursor theme manually with gdm 3.20.x? I think the information Arch Linux wiki on this is outdated

**Comment by sljunkie** [comment]
wait a minute, GDM 3.20.1-2 is out on Arch.

Here are the [changes in the PKGBUILD](https://git.archlinux.org/svntogit/packages.git/commit/trunk?h=packages/gdm&id=ff1a0205d6b2cb2ed2d687fe094a557aafc2b89e)

**Comment by Ste74** [comment]
At the moment i have stopped to use gdm because in my laptop have unresolved issue :  suspend on lid not work ( and yes i have try any solution found on the web ) and random freeze on desktop .. I have switched on lightdm ( plus gnome screensaver ) and all work very well ..

**Comment by philmmanjaro** [comment]
@sljunkie: you can find it in our **unstable** branch.

**Comment by Ste74** [comment]
@sljunkie can be closed this ?

**Comment by sljunkie** [comment]
@Ste74 if gdm isn't installed anymore on a fresh Manjaro Gnome installation, yes.

the gdm3setup utility is obsolete. I'm still using gdm nonetheless, but I had to dig trough the documentation to change the icon and cursor theme.

See [closed upstream bug #775987](https://bugzilla.gnome.org/show_bug.cgi?id=775987)
